### ISSUE TITLE
Incorrect error reporting with requests package

### ISSUE THREAD

**Issue by elirnm** [PyCQA/pylint#1411]
### Current behavior
Using the requests package, `requests.codes.ok` reports `error,E1101:Instance of LookupDict' has no 'ok' member` even thought it's a valid, functioning line of code.

### pylint --version output
```No config file found, using default configuration
pylint 1.6.5,
astroid 1.4.9
Python 3.4.3 (v3.4.3:9b73f1c3e601, Feb 24 2015, 22:43:06) [MSC v.1600 32 bit (Intel)]
```

**Comment by rogalski** [comment]
[Attributes are set automagically](https://github.com/kennethreitz/requests/blob/master/requests/status_codes.py) - I guess we need an astroid brain hint (or try to understand `setattr`calls - but that seems like a non-zero effort.

**Comment by PCManticore** [comment]
We're not going to support every library out there with brain plugins unfortunately. Feel free to use ``generated-members`` or other tricks for supporting dynamic attributes, as seen in the documentation over here https://pylint.readthedocs.io/en/latest/technical_reference/features.html#id28.

**Comment by jeppoo1** [comment]
Still having this issue on 20.11.2019, Python 3.7.4, requests 2.22.0., VSCode Insiders.

It does not show an error on every requests.xxxxx -part, only some of them (my error is at requests.codes.ok),

**Comment by andrei-korshikov** [comment]
@jeppoo1 That's expected behavior because of dynamic nature of `requests.codes`. The solution is using `generated-members=requests.codes.ok` configuration option, as @PCManticore clearly stated above. The same is written in [no-member / E1101](https://pylint.readthedocs.io/en/latest/user_guide/messages/error/no-member.html) pylint doc page at the very bottom.
